src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -401,4 +410,14 @@ private function checkUserAction(Entry $entry)
             throw $this->createAccessDeniedException('You can not access this entry.');
         }
     }
+
+    /**
+     * @return int
+     */
+    private function _countAllEntriesForUser()
+    {
+        $repository = $this->getDoctrine()->getRepository('WallabagCoreBundle:Entry');
+
+        return $repository->countAllEntriesByUsername($this->getUser()->getId());
```

Comment:
Since you are using that method only once, you can move it inside `showUnreadAction`